marklogic-data-hub/src/main/java/com/marklogic/hub/flow/impl/FlowRunnerImpl.java

```diff
@@ -55,7 +55,7 @@
 
     private StepRunner stepRunner;
 
-    private final Map<String, Queue<String>> stepsMap = new ConcurrentHashMap<>();
+    private final ConcurrentHashMap<String, Queue<String>> stepsMap = new ConcurrentHashMap<>();
```

Comment:
Can you include a brief comment to explain why this map has a concrete type but the others are just Map? Otherwise some well-meaning developer in the future will change this to Map to be consistent with the others.

Reply:
Added the comment